### PR TITLE
[v0.6.8] tx broadcast debugging

### PR DESCRIPTION
**Pull request by jchappelow**
enabling this for CI

### CODE REVIEW

File: cmd/kwil-cli/cmds/account/balance.go [hunk 1]
```diff
@@ -41,7 +41,7 @@ func balanceCmd() *cobra.Command {
 						return display.PrintErr(cmd, errors.New("empty account ID"))
 					}
 				}
-				acct, err := cl.GetAccount(ctx, acctID, types.AccountStatusLatest)
+				acct, err := cl.GetAccount(ctx, acctID, types.AccountStatusPending)
```
[the remark this answers is not in the document]
Comment (charithabandi) [reply cmd/kwil-cli/cmds/account/balance.go:44]: So that we have both the info

[the remark this answers is not in the document]
Comment (brennanjl) [reply cmd/kwil-cli/cmds/account/balance.go:44]: I think that makes sense. A `--confirmed` nonce or something

[the remark this answers is not in the document]
Comment (jchappelow) [reply cmd/kwil-cli/cmds/account/balance.go:44]: k will add a flag.  Long ago I wanted it to show both, but that's more work for server I guess.

[the remark this answers is not in the document]
Comment (jchappelow) [reply cmd/kwil-cli/cmds/account/balance.go:44]: Oh, but what should default be?

[the remark this answers is not in the document]
Comment (jchappelow) [reply cmd/kwil-cli/cmds/account/balance.go:44]: Um, nvm, default would be pending if flag is `--confirmed`

[the remark this answers is not in the document]
Comment (jchappelow) [reply cmd/kwil-cli/cmds/account/balance.go]: OK `--confirmed` added to the `account balance` command's flags.  Also in https://github.com/kwilteam/kwil-db/pull/682
Its a change of default for kwil-cli, which I think is fine personally, but we can reverse the semantics with a `--pending` flag if we have concerns.

[the remark this answers is not in the document]
Comment (charithabandi) [reply cmd/kwil-cli/cmds/account/balance.go]: Yeah probably --pending is a good idea

[the remark this answers is not in the document]
Comment (jchappelow) [reply cmd/kwil-cli/cmds/account/balance.go]: Haha, what's the verdict?  I wonder if pkoch assumed it would report pending...

**Review by charithabandi — APPROVED** [review]

**Pull request merged**